Source/DafnyCore/AST/MemberDecls.cs

```diff
@@ -97,6 +97,7 @@ public class Field : MemberDecl, ICanFormat, IHasDocstring {
   public override string WhatKind => "field";
   public readonly bool IsMutable;  // says whether or not the field can ever change values
   public readonly bool IsUserMutable;  // says whether or not code is allowed to assign to the field (IsUserMutable implies IsMutable)
+  public PreType PreType;
```

Comment:
Why is `Type` readonly, but `PreType` isn't?

Reply:
Same as for `ResultPreType`, see above.